qa/automation/ScoringPortal.Tests.Scenarios/terenteva/sign_in_with_Google_account.feature

```diff
@@ -0,0 +1,91 @@
+Feature: 1. Sign in to Quantori Scoring Portal with Google account feature
+
+ Background:
+ 	Given Quantori Scoring Portal page (QSPP) exists
+ 	And QSPP is linked to the Internet address https://qa.ddso-spot.quantori.com/
+ 	And Two versions of QSPP could be displayed by the same address: QSPP 'User NOT signed in' and QSPP 'User signed in'
+ 	And On the QSPP 'User NOT signed in' version following components are displayed:
+ 		| Component										|
+ 		| 'Sign in with Google' button					|
+ 		| Quantori Scoring	logotype image				|
+ 	And On the QSPP 'User signed in' version following components are displayed:
+ 		| Component                                     |
+ 		| Quantori Scoring	logotype image              |
+		| Divider                                       |
+ 		| 'Dataset' text line                           |
+ 		| Divider                                       |
+ 		| Drag & drop form								|
+ 		| Username text field							|
+ 		| User profile icon								|
+ 		| Sign out button with sign out icon			|
+
+ Scenario: 1_01. 'Sign in with Google' button on the QSPP 'User NOT signed in' version is interactive and hyperlinked
+	Given QSPP 'User NOT signed in' version is opened in a browser tab
+	When User drags cursor to the 'Sign in with Google' button
+	Then 'Sign in with Google' button is highlighted
+	And Cursor appearance is changed from standard to Link pointer
+	And Hyperlink address is shown at the bottom left corner of the page
+
+ Scenario Outline: 1_02. User is redirected to Google Login page
+ 	Given QSPP 'User NOT signed in' version is opened in a browser tab
+	And User Google account state is <User account state>
+	When User clicks on 'Sign in with Google' button
+	Then User is redirected to Google Accounts Login page
+	And Google Accounts Login page is opened in the same browser tab
+
+	  Examples: 
+	  | User account state                                     |
+	  | User has no Google account                             |
+	  | User has a Google account, is not signed in the account |
+	  | User has a Google account, is signed in it, not signed in QSPP before from the device and browser used |
+
+ Scenario: 1_03. User goes straight from QSPP 'User not signed in' to QSPP 'User signed in'
+	Given QSPP 'User NOT signed in' version is opened in a browser tab
+	And User has a Google account
+	And User is signed in this Google account
+	And User has signed in to QSPP before from the device and browser used
+	When User clicks on 'Sign in with Google' button
```

Comment:
User can have several pages (e.g. app under test and some online shop  with  "sign in with Google" button) open in browser , which page is user on? Can this info be put in Background?

Reply:
I think, I would add 'current' to where browser tab user is at is mentioned, would that be ok with you? I don't think it could be added to Background, because there are two page versions user might be seeing in different Givens.

Reply:
why not write 'When user clicks on 'Sign in with Google' button on QSPP'? You can also add which QSPP version is user on

Reply:
I think' I'll go with 'User is on QSPP ... version' variant for every Scenario, because 'user clicks' is usually in When section, and I think, which version of QSPP user is currently on should be mentioned in Given.